doc/_toc.yml

```diff
@@ -9,10 +9,9 @@ parts:
     - file: intro
     - file: connecting
     - file: plot
-    - file: duckdb
```

Comment:
Why those changed?

Reply:
Those are the ones you moved to `integrations/`, so they're duplicated. the files are no longer there so they don't show up in the docs and should be removed